app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala

```diff
@@ -89,17 +89,23 @@ trait IncomeUpdateCalculatorController extends TaiBaseController
         }
   }
 
-
   def howToUpdatePage(id: Int): Action[AnyContent] = authorisedForTai(personService).async { implicit user =>
     implicit person =>
       implicit request =>
 
         employmentService.employment(Nino(user.getNino), id) flatMap {
-          case Some(employment) =>
+          case Some(employment: Employment) =>
+
+            val incomeType = incomeTypeIdentifier(employment.receivingOccupationalPension)
+
             for {
-              incomeToEdit <- incomeService.employmentAmount(Nino(user.getNino), id)
+              incomeToEdit: EmploymentAmount <- incomeService.employmentAmount(Nino(user.getNino), id)
```

Comment:
Can these be ran concurrently?

Reply:
They can. All I added was a type though :(

Reply:
Sorry chap. This was meant to be a 'as you are in the area how about this?'. Perhaps there is a different way to flag these.